integration-tests/main-app-test/src/test/java/co/elastic/apm/test/AgentSetupIT.java

```diff
@@ -61,9 +62,9 @@ void testServiceNameAndVersionFromManifest(String image, AgentFileAccessor.Varia
     void testSecurityManagerWarning() {
         String expectedMsg = "Security manager without agent grant-all permission";
 
-        try (TestAppContainer app = testAppWithJavaAgent("openjdk:17", AgentFileAccessor.Variant.STANDARD)) {
+        try (AgentTestContainer.Generic app = testAppWithJavaAgent("openjdk:17", AgentFileAccessor.Variant.STANDARD)) {
 
-            app.withSecurityManager()
+            app.withSecurityManager(null)
```

Comment:
[for reviewer] using `null` here is a mean to indicate that we just enable the SM with the default policy.